.github/workflows/deploy.yml

```
@@ -12,7 +12,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+    - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4
```

If using digest pinning, specify the patch version in the comments to make it easier to understand Renovate's updated PRs. Renovate will automatically update this comment and display the changelog between versions.
```suggestion
    - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.1
```

---

Thanks! I went ahead and updated it to match the current version (4.1.4).